.buildkite/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:22.04
+ARG DEBIAN_FRONTEND=noninteractive
+
+ENV DA_AWS_ACCESS_KEY_ID="default"
+ENV DA_AWS_SECRET_ACCESS_KEY="default"
+ENV AWS_DEFAULT_REGION="default"
+
+# Install the good stuff
+RUN apt-get update
+
+RUN apt install unzip -y
+RUN apt install wget -y
+RUN apt install curl -y
+
+# Install Git
+RUN apt-get install -y git
+
+# Install Ansible
+RUN apt-get install -y software-properties-common && \
+    apt-add-repository --yes --update ppa:ansible/ansible && \
+    apt-get install -y ansible
```

Comment:
[Question] Why use `apt install` in some and `apt-get install` in other layers? Curious.

[Suggestion] Can we merge the RUN layers ?

```
RUN apt-get update && apt-get install -y \
  curl \
  git \
  software-properties-common \
  ...
  // and possibly remove /var/lib/apt/lists/*
```

Reply:
I mostly used whatever was in the doc for each vendor. I just checked and merging the layers cut the size by about 10mb on an image that is fairly chonky at 750mb (using ubuntu to match the servers). I'll add it in.

Reply:
Nice, it's something :smile: